### DISCUSSION TITLE
Feature request: Plex movie tag: "Edition"

### DISCUSSION THREAD

**Discussion by hamakavula** [croneter/PlexKodiConnect#1876] [Ideas]
Hi all,

Recently PMS has added the movie option "Edition" for each title.

![image](https://user-images.githubusercontent.com/17685480/199301290-ba16eb01-08a0-49eb-b022-80b47bdef1da.png)

This is handy because now this 'tag' doesn't need to be in the title and gets displayed below the title (like movie-release-year already did).

![image](https://user-images.githubusercontent.com/17685480/199301543-6386706a-e420-4c51-81c2-f0d446f1ab5b.png)

**This 'tag' doesn't show up in PKC on Kodi but would be an awesome feature request!**

**Comment by croneter** [comment]
Is there a similar "Edition" element in the Kodi vanilla gui I could use for that? I don't think so. Any suggestions how I could implement this?

**Reply by hamakavula** [reply]
I don't think so either but behind the title would be the best option. Like you did with 'movie release year'.

I name files the same way "TITLE (YYYY) EDITION.ext"

**Comment by RickSWilliams** [comment]
Just curious how this should be behaving now? I'm new to Kodi, but it seems to have support for editions (versions as Kodi calls them). So far PKC doesn't seem to be utilizing that feature at all. Just want to make sure I am not missing some basic setting. Thanks for the work on the add-on. It has been fantastic.

I am able to assign editions through the Kodi UI, but it’s a manual process. I’m also having a hard time getting both editions to download the extra artwork. It seems like only the first version in the Plex sort order gets extra artwork types downloaded. 

I also had to switch to direct mode to get the files to play after being grouped. Add-On mode would cause an error. Let me know if I can help with any more details.  

The best default behavior would just be to assign an edition name in Kodi that matches the one assigned in Plex, but not group the movies as a single entry. Plex doesn't group them either, so it would be consistent.
